src/transferslot.cpp

```diff
@@ -1083,7 +1097,21 @@ void TransferSlot::doio(MegaClient* client)
         }
     }
 
-    p += transfer->progresscompleted;
+    if (transfer->type == GET && transferbuf.isRaid())
```

Comment:
Since we don't put this stuff into functions can we leave comments as to what the individual blocks of code do?

Reply:
comment added :-)